app/controllers/pages_controller.rb

```diff
@@ -2,7 +2,14 @@ class PagesController < ApplicationController
   layout "full-width"
   before_action :redirect_to_dashboard, only: [:login]
 
+  RETIRED_SLUGS = {
+    "supporting-partners" => :get_involved_path,
+    "contributing-partners" => :get_involved_path
+  }.freeze
+
   def page
+    redirect_to send(RETIRED_SLUGS[params[:page_slug]]) and return if RETIRED_SLUGS.key?(params[:page_slug])
+
```

Comment:
This is a bit interesting, as I think there must have been retirements and redirects done previously? This works, I just wonder if there are any previous solutions / ways of dealing with these retire and redirect scenarios.

Reply:
Thanks for this! I moved the redirects into the routes file